substrate/frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -709,6 +715,9 @@ pub mod pallet {
 		/// The configuration of benchmarking.
 		type BenchmarkingConfig: BenchmarkingConfig;
 
+		/// Provider for the block number. Normally this is the `frame_system` pallet.
+		type BlockNumberProvider: BlockNumberProvider;
```

Comment:
what is normally? prdoc is read once, the config doc read by all new clients

Reply:
i have doc properly now